### PR TITLE
fix Cannot read property 'type' of undefined

### LINKED ISSUE #5: Cannot read property 'type' of undefined
I'm getting a type of undefined on this line
node_modules/eslint-plugin-knex/rules/avoid-injections.js:30:15

code triggering the issue
```
const sharp = require('sharp');

sharp(input).raw();
```

### PR DESCRIPTION
**Pull request by mahcloud**
fixes #5

### CODE REVIEW

**Comment by mahcloud** [comment]
@AntonNiklasson what do you think about this fix?

File: rules/avoid-injections.test.js [hunk 1]
```diff
@@ -26,6 +26,13 @@ ruleTester.run("avoid-injections", rule, {
     "knex('users').whereRaw('id = ?', [1]);",
     "knex('users').whereRaw(`id = 1`);",
     "const joinCondition = `blog_posts ON users.id = blog_posts.author`; knex('users').select(['email']).joinRaw(joinCondition)",
+    `function thing1() {
+      return {
+        thing2: () => {},
+      };
+    }
+    thing1().thing2();
```
Comment (AntonNiklasson) [inline rules/avoid-injections.test.js]: This code is already passing, without the change. Can we add the snippet from the issue here instead?

File: rules/avoid-injections.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ function check(context, node) {
   const statement = node.callee.property.name;
   const queryNode = node.arguments[0];
 
+  if (queryNode === undefined) return;
```
Comment (AntonNiklasson) [inline rules/avoid-injections.js]: I would just make this part of the early return below.

**Review by AntonNiklasson — CHANGES_REQUESTED** [review]
Thanks a lot for taking the time to work on this! A few minor things.

File: rules/avoid-injections.js (hunk 2 above)
Comment (mahcloud) [reply rules/avoid-injections.js]: update in https://github.com/AntonNiklasson/eslint-plugin-knex/pull/6/commits/add9101aec13ea040fd67548daeeff1c52e34910

File: rules/avoid-injections.test.js (hunk 1 above)
Comment (mahcloud) [reply rules/avoid-injections.test.js]: updated in https://github.com/AntonNiklasson/eslint-plugin-knex/pull/6/commits/add9101aec13ea040fd67548daeeff1c52e34910

**Comment by mahcloud** [comment]
@AntonNiklasson ready for another review please

**Review by AntonNiklasson — APPROVED** [review]
Perfect! Thanks a lot 🥇

**Comment by AntonNiklasson** [comment]
This is included in `v1.0.4` 🎉

**Pull request merged**